modules/Standardization/src/client/Standardization.coffee

```diff
@@ -19,19 +19,49 @@ class StandardizationCurrentSettingsController extends Backbone.View
 				@$('.bv_getCurrentSettingsError').show()
 
 	setupCurrentSettingsTable: (settings) ->
+
+		if settings.invalidReasons? && settings.invalidReasons != ""
+			# Convert new line seperated string into ul list
+			invalidReasonsHtml = settings.invalidReasons.replace(/\n/g, "</li><li>")
+			invalidReasonsHtml = "<ul><li>" + invalidReasonsHtml + "</li></ul>"
+		else
+			invalidReasonsHtml = null
+
+		if settings.needsRestandardizationReasons? && settings.needsRestandardizationReasons != ""
+			# Convert new line seperated string into ul list
+			needsRestandardizationReasonsHtml = settings.needsRestandardizationReasons.replace(/\n/g, "</li><li>")
+			needsRestandardizationReasonsHtml = "<ul><li>" + needsRestandardizationReasonsHtml + "</li></ul>"
+			needsStandardizationReasons = settings.needsRestandardizationReasons
+		else
+			needsRestandardizationReasonsHtml = null
+			needsStandardizationReasons = null
+
+		if settings.suggestedConfigurationChanges? && settings.suggestedConfigurationChanges != ""
+			msg = "Your standardizer settings are incomplete and have been automatically interpreted.<br>Please update your standardizer configuration with the following to avoid ambiguity:"
+			suggestedConfigurationChangesHtml = settings.suggestedConfigurationChanges.replace(/\n/g, "</li><li>")
+			suggestedConfigurationChangesHtml = "<ul><li>" + suggestedConfigurationChangesHtml + "</li></ul>"
+			suggestedConfigurationChangesHtml = "<b>#{msg}</b><br>#{suggestedConfigurationChangesHtml}"
+		else
+			suggestedConfigurationChangesHtml = null
+	
 		@$('.bv_currentSettingsTable').dataTable
 			"aaData": [
+				[ "Settings valid", settings.valid],
+				[ "Settings invalid reasons", invalidReasonsHtml],
```

Comment:
I just added those suggested changes. I also removed the Time modified since it could be confusing. Going to re-request approval because I want to make sure what I did matches your suggestions.